source/content/guides/migrate/acquia/05-migrate.md

```diff
@@ -14,7 +14,7 @@ layout: guide
 permalink: docs/guides/acquia/migrate
 anchorid: migrate
 editpath: migrate/acquia/05-migrate.md
-reviewed: "2021-06-15"
+reviewed: "2022-12-13"
 ---
 
 Pantheon provides an Acquia path for migrating existing sites to the platform. This process begins by clicking the **Migrate Existing Site** button on the User Dashboard.
```

Comment:
@stovak , is step 3 correct?  see https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/acquia/migrate